compiler/compiler/src/main/java/org/robovm/compiler/target/framework/FrameworkTarget.java

```diff
@@ -120,12 +120,15 @@ public String getInstallRelativeArchivePath(Path path) {
 	protected List<String> getTargetExportedSymbols() {
 		return Arrays.asList("JNI_*", "rvmInstantiateFramework");
 	}
-	
-	private String getMinimumOSVersion() {
-		NSObject minimumOSVersion = config.getInfoPList().getDictionary().objectForKey("MinimumOSVersion");
```

Comment:
please use getInfoPList() instead of getIosInfoPList() to keep it consistent in this class. 
no need to check getInfoPList for null as it is mandatory for this target (check init)

Reply:
Ok, I've just copied the IOSTarget implementation to FrameworkTarget, Will roll it back